app/src/test/kotlin/com/intive/picover/ExampleUnitTest.kt

```diff
@@ -1,16 +1,11 @@
 package com.intive.picover
 
-import org.junit.Assert.assertEquals
-import org.junit.Test
+import io.kotest.core.spec.style.ShouldSpec
+import io.kotest.matchers.shouldBe
 
-/**
- * Example local unit test, which will execute on the development machine (host).
- *
- * See [testing documentation](http://d.android.com/tools/testing).
- */
-class ExampleUnitTest {
-	@Test
-	fun addition_isCorrect() {
-		assertEquals(4, 2 + 2)
+class ExampleUnitTest : ShouldSpec({
```

Comment:
Migrated just only to keep working exampled. It should be removed when first "regular" unit test will be added